src/elements/content-picker/ContentPicker.js

```diff
@@ -206,6 +207,8 @@ class ContentPicker extends Component<Props, State> {
             isLoading: false,
             errorCode: '',
         };
+
+        setElementsVersion();
```

Comment:
This would imply that, if we want to add additional items to the global object, we'd need one function call per line. Would something like `globals.init()` give us more breathing space?

Reply:
I left it as a specific method in case different consumers only care about certain properties.